src/program.js

```diff
@@ -24,7 +25,11 @@ export class Program {
 
   constructor(
     argv: ?Array<string>,
-    {absolutePackageDir = process.cwd()}: {absolutePackageDir?: string} = {}
+    {
+      absolutePackageDir = process.cwd(),
+    }: {
+      absolutePackageDir?: string,
```

Comment:
even though there is only one param, I think it's worth moving this to a standalone type object like we do elsewhere:
````javascript
type ProgramOptions = {
  absolutePackageDir?: string,
}
````